app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,17 @@
     <uses-permission android:name="android.permission.VIBRATE"/>
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+    <uses-permission android:name="com.myapp.android.locationapi.maps.permission.MAPS_RECIEVE"/>
```

Comment:
Potentially useless permission.  @ayoung1 can we verify if the locationapi permission is needed.  According to [this](http://stackoverflow.com/questions/14832911/android-map-v2-why-maps-receive-permission) it is not.